src/components/Input/input.scss

```diff
@@ -19,11 +19,15 @@ input::placeholder {
   color: $hint;
   font-size: 18px;
 }
-
+.input-wrapper{
+  display: flex;
+  flex-direction: column;
+  height: 80px;
```

Comment:
Try to avoid fixed heights and use flex more. If that's really necessary, use rems - this is 4.44rem. Why should this be fixed height?